pkg/collectsub/collectsub/collectsub.pb.go

```diff
@@ -98,6 +98,7 @@ type CollectEntry struct {
 
 	Type  CollectDataType `protobuf:"varint,1,opt,name=type,proto3,enum=guacsec.guac.collect_subscriber.schema.CollectDataType" json:"type,omitempty"`
 	Value string          `protobuf:"bytes,2,opt,name=value,proto3" json:"value,omitempty"`
+	AsOf  int64           `protobuf:"int64,3,opt,name=as_of,proto3" json:"as_of,omitempty"`
```

Comment:
this file should be accompanied with the collectsub.proto file change and this change should propagate with running  `make proto`. If you don't have the proto utils set up i can help you generate the code and send it over as well.

Reply:
I don't think there is a need to change this as it is already part of the filters.

Reply:
Installed and `make proto`'d now.

Reply:
yeah, there is no reason to change the proto file. It already had the time value, just not being utilized.

Reply:
yea i think the question is whether `asOf` wants to be part of the proto, which then if the time it was added wants to be returned, it would need to be changed in [CollectEntry](https://github.com/guacsec/guac/blob/main/pkg/collectsub/collectsub/collectsub.proto#L30)